src/main/java/baritone/altoclef/AltoClefSettings.java

```diff
@@ -76,6 +76,20 @@ public void avoidBlockBreak(Predicate<BlockPos> avoider) {
         }
     }
 
+    /**
+     * If the original reference is passed and in the list,
+     * it will be removed from it.
+     *
+     * @param avoider original object reference that is in the list.
+     */
+    public void removeAvoidBlockBreak(Predicate<BlockPos> avoider) {
```

Comment:
Does this even work? I'm pretty sure functions can't be referenced/accessed like this. I'd also advise against having this in here since `BotBehaviour`'s `push` and `pop` functions already deal with this.